tron/core/job.py

```diff
@@ -176,12 +180,22 @@ def state_data(self):
         """This data is used to serialize the state of this job."""
         return {
             'runs': self.runs.state_data,
-            'enabled': self.enabled,
+            'enabled': {
```

Comment:
is this data used in  tronview / tronweb? the change in structure will have to be reflected.

Reply:
No, state_data is only used during serialization to tron_state. tronview and tronweb, by querying the api, only get access to the enabled attribute in a Job object.